homeassistant/components/systemmonitor/util.py

```diff
@@ -25,6 +25,11 @@ def get_all_disk_mounts() -> set[str]:
                 "No permission for running user to access %s", part.mountpoint
             )
             continue
+        except OSError as err:
+            _LOGGER.warning(
+                "Mountpoint %s was excluded because of: %s", part.mountpoint, str(err)
```

Comment:
Every exception has a `__str__` magic method.
```suggestion
                "Mountpoint %s was excluded because of: %s", part.mountpoint, err
```